backend/LexBoxApi/GraphQL/UserMutations.cs

```diff
@@ -96,15 +96,16 @@ IEmailService emailService
     [Error<DbError>]
     [Error<UniqueValueException>]
     [Error<RequiredException>]
-    [AdminRequired]
     public async Task<LexAuthUser> CreateGuestUserByAdmin(
+        IPermissionService permissionService,
         LoggedInContext loggedInContext,
         CreateGuestUserByAdminInput input,
         LexBoxDbContext dbContext,
         IEmailService emailService
     )
     {
         using var createGuestUserActivity = LexBoxActivitySource.Get().StartActivity("CreateGuestUser");
+        permissionService.AssertCanCreateGuestUserInAnyProject();
```

Comment:
Assuming we
1) change the "AnyProject" check to restrict it to system admins
2) Extend this mutation to accept an org ID to add the new user to (which I think makes sense)
Then...
If org-ID is specificed we need to make sure the current user is a manager of that org.

Reply:
Done in commit a1de5f1b5.